app/models/document.rb

```diff
@@ -32,7 +32,7 @@ class Document < ApplicationRecord
   scope :of_type, ->(type) { where(document_type: type) }
 
   belongs_to :intake, optional: true
-  belongs_to :client, optional: true
+  belongs_to :client, optional: true, touch: true
```

Comment:
Cool! I didn't know you could tell an object to touch an associated object automatically!